helm_deploy/values-preprod.yaml

```diff
@@ -1,37 +1,18 @@
+generic-service:
+  springProfiles: ""
```

Comment:
I can see this is the same as what was there previously but should be set to sqs-read, same for prod 🤔  Ah! [This PR](https://github.com/ministryofjustice/court-list-splitter/pull/18) should have been merged 😬  

Could you add that profile to preprod and prod as part of this deployment as that other PR will be stale now? I'll close the other one.

Reply:
Yep sure no probs